go.sum

```diff
@@ -109,6 +109,8 @@ github.com/DataDog/agent-payload v4.63.1-0.20210329150802-e96c27c6f3cc+incompati
 github.com/DataDog/agent-payload v4.63.1-0.20210329150802-e96c27c6f3cc+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
 github.com/DataDog/agent-payload v4.63.1-0.20210331191159-fca851b5a4f8+incompatible h1:X39+EmZCL3VdM5PPz2bzHbRq4Gmn+G+5ZsLeECgJYAU=
 github.com/DataDog/agent-payload v4.63.1-0.20210331191159-fca851b5a4f8+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.63.1-0.20210401142704-3f5b07c875af+incompatible h1:3myTRQgTdLRZpAlk96PHdCVZxYlJAnp5/4K+QHZoh+M=
+github.com/DataDog/agent-payload v4.63.1-0.20210401142704-3f5b07c875af+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
```

Comment:
Can you try running `go mod tidy`? That should get rid of the older versions.

Reply:
Sure! will do once I merge the `agent-payload` PR and update this one